cleanup/action.yml

```diff
@@ -1,11 +1,14 @@
 ---
 name: 'Clean workspace'
-description: 'Clean workspace directory after previous workflow run'
+description: 'Clean workspace directory by making an empty detached commit'
 runs:
   using: 'composite'
   steps:
     - run: |
         set -e
         shopt -s dotglob
-        rm -rf ${{ github.workspace }}/*
+        git checkout -f \
+          $(git -c user.name=TarantoolBot -c user.email=bot@tarantool.io commit-tree \
+          $(git hash-object -t tree /dev/null) -m 'empty commit' \
```

Comment:
Could you please capitalize the commit message?

Reply:
done